tests/FSharp.Compiler.ComponentTests/Language/InterpolatedStringsTests.fs

```diff
@@ -128,4 +128,34 @@ type Foo () =
         x
         """
         |> compile
-        |> shouldSucceed
\ No newline at end of file
+        |> shouldSucceed
+
+    [<Theory>]
+    // Test different number of interpolated string parts
+    [<InlineData("$\"\"\"abc{\"d\"}e\"\"\"", "abcde")>]
```

Comment:
Well if the result is the same for all the theory cases, you don't have to put it in the theory cases :)

Reply:
That's true, I expected there's going to be more different examples... and actually perhaps there should at least be a multiline one just in case.